### PR TITLE
Soulmates merchandising rebranding

### PR DESCRIPTION
**Pull request by tijanamiletic**
Update to previous set of design changes: https://github.com/guardian/frontend/pull/9692

Wide:
screen shot 2015-08-18 at 16 54 33

Desktop:
screen shot 2015-08-18 at 16 54 43

Ipad landscape:
screen shot 2015-08-18 at 16 54 54

Ipad portait:
screen shot 2015-08-18 at 16 55 06

Mobile landscape:
screen shot 2015-08-18 at 16 55 30

Mobile portrait:
screen shot 2015-08-18 at 16 55 55

### CODE REVIEW

File: static/src/stylesheets/module/commercial/_soulmates.scss [hunk 1]
```diff
@@ -468,31 +479,22 @@ $c-soulmates-high: colour(neutral-1);
...
             @include mq(320px, $until: 479px) {
                 width: 90px;
-                &:last-child {
-                    line-height: 1.2;
-                }
             }
 
             @include mq(leftCol, $until: wide) {
                 width: 90px;
+
```
Comment (Calanthe) [inline static/src/stylesheets/module/commercial/_soulmates.scss]: empty line?

**Comment by tijanamiletic** [comment]
Empty line removed

**Comment by tijanamiletic** [comment]
Refactored css to remove '!important'

**Comment by uplne** [comment]
:+1:

**Pull request merged**